tests/integration/command-functions.test.ts

```diff
@@ -1,26 +1,100 @@
+import processesExample from '../fixtures/processes.json' assert { type: 'json' }
```

Comment:
depcheck doesn't support import assert and is therefore bugging out here. Move this import to another `.ts` file and re-export. We had to do something similar here https://github.com/digicatapult/dscp-api/blob/main/app/version.js

Reply:
got ya. this should be resolved and also removed from `...test.ts` file